A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1335,30 +1336,8 @@ class FMCMainDisplay extends BaseAirliners {
         }
     }
 
-    getCleanApproachSpeed() {
```

Comment:
This calculation from Asobo is bogus, and does not match Airbus docs (clean approach speed is always simply green dot speed).